src/internet_identity/src/anchor_management/registration/registration_flow_v2.rs

```diff
@@ -38,17 +52,40 @@ pub async fn identity_registration_start() -> Result<IdRegNextStepResult, IdRegS
         return Err(IdRegStartError::InvalidCaller);
     }
 
-    let (captcha_png_base64, flow_state) = captcha_flow_state(time()).await;
+    let flow_state = match captcha_required() {
+        true => captcha_flow_state(time()).await.1,
```

Comment:
Does it make sense to create a `let now = time();` variable? Or there is no need?

Reply:
Ah, yes. Thanks.